testsuite/model/test-all-profiles.sh

```diff
@@ -11,11 +11,11 @@ mvn -version
 
 EXIT_CODE=0
 mvn clean
-for I in `perl -ne 'print "$1\n" if (m,<id>([^.<]+)</id>,)' pom.xml`; do
+for I in `perl -ne 'print "$1\n" if (m,<id>([^.<]+)</id>,)' pom.xml | grep offline`; do
```

Comment:
I guess, this change is not desired?

Reply:
This was unintended, thank you for spotting this.